modelci/runs.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
I recommend using something like `ContainerManager` as a class here, and merge the `docker_api_utils.py` with this file.